harness/config/commands.yml

```diff
@@ -186,6 +186,10 @@ command('recompile'):
     DOCKER_BUILDKIT: "1"
   exec: |
     #!bash(workspace:/)|@
+    if docker-compose ps app | grep -q 'exited'; then
```

Comment:
this doesn't work on docker-compose v1

Reply:
sadly, I was hoping for `[ -z "$(docker-compose ps --filter status=running -q app)" ]` to work, but the filter doesn't. That would be the portable way if it worked

Reply:
```suggestion
    if ! docker ps -q -f "status=running" --no-trunc | grep -Fxe "$(docker-compose ps -q console)" >/dev/null then
```

Reply:
basically don't rely on human readable text to be backwards compatible

Reply:
tho if we can say everyone has to have compose v2 installed as `docker compose`, then:
```
if ! docker compose ps -q --status running app >/dev/null; then
```

Reply:
Nice - I've committed your suggestion above there which seems to work :) Thanks Andy!